src/configuration_env.js

```diff
@@ -13,12 +13,32 @@
 import Configuration from './configuration.js';
 import Utils from './util.js';
 
+const deprecatedEnvVars = {
+  JAEGER_SAMPLER_HOST: 'JAEGER_SAMPLER_MANAGER_HOST_PORT',
+  JAEGER_SAMPLER_PORT: 'JAEGER_SAMPLER_MANAGER_HOST_PORT',
+  JAEGER_REPORTER_ENDPOINT: 'JAEGER_ENDPOINT',
+  JAEGER_REPORTER_USER: 'JAEGER_USER',
+  JAEGER_REPORTER_PASSWORD: 'JAEGER_PASSWORD',
+  JAEGER_REPORTER_AGENT_HOST: 'JAEGER_AGENT_HOST',
+  JAEGER_REPORTER_AGENT_PORT: 'JAEGER_AGENT_PORT',
+  JAEGER_DISABLE: 'JAEGER_DISABLED'
+};
+
 export default class ConfigurationEnv {
   static _getConfigValue(obj, key, defaultValue) {
     return (obj && obj[key]) || defaultValue;
   }
 
   static _getSamplerFromEnv(config) {
+
+    Object
```

Comment:
why is this check inside `_getSamplerFromEnv` function?

Reply:
Node.js gets env variables in runtime (call of `getenv` for each variable). So check is when process.env is used.

Related issues:
- https://github.com/nodejs/node/issues/3104
- https://github.com/nodejs/node/issues/22960